packages/support/test/e2e/image-util.e2e.spec.js

```diff
@@ -1,4 +1,4 @@
-import { cropBase64Image } from '../../lib/image-util';
+import {cropBase64Image} from '../../lib/image-util';
```

Comment:
This change seems to be done by the linter, should I keep it or remove it? 
@jlipps

Reply:
yeah that's ok, this file probably hasn't been autoformatted by prettier yet